### DISCUSSION TITLE
Webhook of Memos

### DISCUSSION THREAD

**Discussion by Zeng1998** [usememos/memos#1037] [Ideas]
Hi all, before implementing webhook in Memos, I would like to discuss its application scenarios with you. 

In other words, like, what will you use Memos webhook for? What kind of webhook is needed in Memos?

**Comment by Zeng1998** [comment]
related issues:  #418 #676 #961

**Reply by longcat99** [reply]
我想通过他来记录短信验证码。
或者通过telegram-bot什么的来发布
I want him to record the text capTcha.
Or publish it via telegram bot or something
![Screenshot_20230222_163330_com idormy sms forwarder](https://user-images.githubusercontent.com/33887488/220565922-3b080bb2-1137-466c-ae6c-0fb1175eddf6.jpg)

**Reply by Zeng1998** [reply]
> 我想通过他来记录短信验证码。 或者通过telegram-bot什么的来发布 I want him to record the text capTcha. Or publish it via telegram bot or something ![Screenshot_20230222_163330_com idormy sms forwarder](https://user-images.githubusercontent.com/33887488/220565922-3b080bb2-1137-466c-ae6c-0fb1175eddf6.jpg)

That's not about webhooks of Memos. You can publish memos via tg bot using open api.

**Comment by sven-debug** [comment]
> Hi all, before implementing webhook in Memos, I would like to discuss its application scenarios with you.
> 
> In other words, like, what will you use Memos webhook for? What kind of webhook is needed in Memos?

I would love to send messages to memos so I could forward AI generated answers automatically to memos to create e new entry. That would be GREAT!

Warm regards
Sven

**Reply by hyponet** [reply]
In fact, I have already implemented similar functionality based on the currently available webhook capabilities.
<img width="2474" height="1640" alt="CleanShot 2025-12-03 at 11 58 22@2x" src="https://github.com/user-attachments/assets/f9176976-de91-4eb3-8878-7d424d5f48cf" />

**Comment by vibbe** [comment]
I would like to use Gotify for this, already have it for when someone post a new memo, but would be good to have it for eg different local server too ip:port, and have it integrated somehow in the UI so its easy to add with different:
* memos
* users (posting memos)
* topics or subject (if post has # subject)
* comments eg in posts
